docker-hub/oci-artifacts.md

```diff
@@ -0,0 +1,175 @@
+---
+title: OCI artifacts on Docker Hub
+keywords: oci, artifacts, docker hub
+description: >
+  You can store any software artifacts on Docker Hub, not just container images
```

Comment:
I would phrase this as either "store OCI artifacts" or "store SW artifacts packaged as OCI images"